charts/postgres-db-manager/templates/grant-access-job.yaml

```
@@ -41,11 +41,14 @@ spec:
             
             if [ "$READ_ONLY" = true ]
             then
-              psql -c "GRANT CONNECT ON DATABASE \"${DB}\" TO \"${USER}\";"
-              psql -d ${DB} -c "GRANT USAGE ON SCHEMA public TO \"${USER}\";"
-              psql -d ${DB} -c "GRANT SELECT ON ALL TABLES IN SCHEMA public TO \"${USER}\";"
-              psql -d ${DB} -c "GRANT SELECT ON ALL SEQUENCES IN SCHEMA public TO \"${USER}\";"
-              echo "${USER} now has readonly access to the database ${DB}"
+              for SCHEMA in ${SCHEMAS}
+              do
+                psql -c "GRANT CONNECT ON DATABASE \"${DB}\" TO \"${USER}\";"
+                psql -d ${DB} -c "GRANT USAGE ON SCHEMA ${SCHEMA} TO \"${USER}\";"
+                psql -d ${DB} -c "GRANT SELECT ON ALL TABLES IN SCHEMA ${SCHEMA} TO \"${USER}\";"
```

Add access to future tables:
```suggestion
                psql -d ${DB} -c "GRANT SELECT ON ALL TABLES IN SCHEMA ${SCHEMA} TO \"${USER}\";"
                psql -d ${DB} -c "ALTER DEFAULT PRIVILEGES IN SCHEMA ${SCHEMA} GRANT SELECT ON TABLES TO \"${USER}\";"     
```